tests/PKSim.Matlab.Tests/MatlabIndividualFactorySpecs.cs

```diff
@@ -63,14 +73,24 @@ public class When_creating_an_individual_based_on_a_valid_origin_data_with_ontog
       protected override void Context()
       {
          base.Context();
-         _matlabOriginData = new OriginData
+         _matlabOriginData = new Core.Snapshots.OriginData
          {
             Species = CoreConstants.Species.HUMAN,
             Population = CoreConstants.Population.ICRP,
-            Age = 30,
-            Weight = 75,
-            Height = 17.5,
-            Gender = CoreConstants.Gender.Male
+            Age = new Parameter
+            {
+               Value = 30
+            },
+            Weight = new Parameter
+            {
+               Value = 75
+            },
+            Height = new Parameter
+            {
+               Value = 17.5,
+               Unit = "dm",
```

Comment:
This is how the unit is specified. This will make scripts much easier to understand